main/get_feature_set.py

```diff
@@ -12,6 +12,11 @@
 G_logs_based = ["rounds", "shepherd_time", "review_time"]
 OSS_logs_based = ["rounds", "shepherd_time"]
 
+drop_cols = ["Indirect_(btw)", "Indirect_(greeting)",
```

Comment:
Can you add a comment about why you're dropping these?

Reply:
added: dropped low importance features to achieve better results